apps/demos/Demos/CardView/CardTemplate/React/VehicleCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useState } from 'react';
+import Button from 'devextreme-react/button';
+import { VehicleCardProps } from './types';
+
+const VehicleCard = ({
+    vehicle,
+    id,
+    model,
+    price,
+    categoryName,
+    modification,
+    bodyStyleName,
+    horsepower,
+    onShowInfo,
+}: VehicleCardProps) => {
+  return (
+    <div className="vehicle__card">
+        <div className="vehicle__img-wrapper">
+            <img className="vehicle__img"
+                src={`../../../../images/vehicles/image_${id}.png`}
+                alt={model}
+            />
+        </div>
+        <div className="vehicle__info">
+            <div className="vehicle__name" title={model}>
+                {model}
+            </div>
+            <div className="vehicle__price">
+                {price}
+            </div>
+            <div className="vehicle__type-container">
+                <div className="vehicle__type">
+                    {categoryName}
+                </div>
+            </div>
+            <div className="vehicle__spec-container">
+                <div className="vehicle__modification">
+                    {modification}
+                </div>
+                <div className="vehicle__modification">
+                    {bodyStyleName}
+                </div>
+                <div className="vehicle__modification">
+                     {horsepower} h.p.
```

Comment:
Let's remove redundant space
```suggestion
                    {horsepower} h.p.
```